src/sentry/static/sentry/app/components/internalLinkButton.jsx

```diff
@@ -0,0 +1,42 @@
+import styled from 'react-emotion';
+import PropTypes from 'prop-types';
+import React from 'react';
+
+import Link from './link';
+import InlineSvg from './inlineSvg';
+
+const InternalLinkButtonText = styled('div')`
+  flex-grow: 1;
+`;
+
+const StyledInlineSvg = styled(InlineSvg)`
+  margin-right: 0.75em;
+`;
+
+const InternalLinkButton = styled(({to, children, icon, ...props}) => (
```

Comment:
We should start thinking about best practices irt emotion.

Reply:
@billyvg yeah i agree. tbh i wasn't just in love with this style of formatting and would be open minded to other ideas.